lib/submission.rb

```diff
@@ -0,0 +1,32 @@
+require 'dotenv/load'
+require 'twilio-ruby'
+
+class Submission
+  def initialize(order = Order.new)
+    @order = order
+
+    error_message = "Add items to order first. Cannot submit empty order."
+    raise error_message if order.total.zero?
+
+    submit_order
+  end
+
+  def submit_order
+    sms_text = "Thank you! Your order was placed at 17:52 and should be with you by 18:52."    
```

Comment:
You can use Time.new to send a msg with the actual time.

Reply:
Good idea